### PR TITLE
693 login form improvements

### LINKED ISSUE #693: LoginAttempts not clearing
We need to investigate the loginattempt system, it appears to not clear.

### PR DESCRIPTION
**Pull request by mauromsl**
Closes #693

### CODE REVIEW

**Comment by mauromsl** [comment]
test this please

File: src/core/forms.py [hunk 1]
```diff
@@ -377,13 +379,38 @@ def clean_email(self):
 
 
 class LoginForm(forms.Form):
-    user_name = forms.CharField(max_length=255)
-    user_pass = forms.CharField(max_length=255, widget=forms.PasswordInput)
+    user_name = forms.CharField(max_length=255, label="Email")
```
Comment (ajrbyers) [inline src/core/forms.py]: Can we make the label a translatable with ugettext?

Comment (mauromsl) [reply src/core/forms.py]: I left the translation logic in the templates as that is what was the current behaviour, but happy to change that as well if you want

Comment (ajrbyers) [reply src/core/forms.py]: Woops, missed that. Its fine as is I think.

**Pull request merged**